test/browser-transmit.test.js

```diff
@@ -253,3 +253,40 @@ test('applies all serializers to messages and bindings (serialize:true)', ({ end
     .fatal({ test: 'test' }, Error())
   end()
 })
+
+test('extracts bindings and raw messages', ({ end, same, is }) => {
+  let messages = null
```

Comment:
```suggestion
  var messages = null
```

(we use var and const in this code base)